2019-03-23 20:47:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "INSERT INTO brand (brand_id, brand_name) VALUES('brand_id','brand_name'),('1','Oreo'),('2','Hoosier HIll Farm'),('3','Tea's Tea'),('4','Smart Ones'),('5','Green Chile'),('6','Vita'),('7','Cut Russet'),('8','Yoplait'),('9','Sanpellegrino'),('10','Minute Maid'),('11','Pepperidge Farm '),('12','Coca Cola'),('13','Cheezit'),('14','Santa Cruz'),('15','Fatworks'),('16','Suprema'),('17','Urban Grain'),('18','Alo Enrich[*]');"; SQL statement:
INSERT INTO brand (brand_id, brand_name) VALUES('brand_id','brand_name'),('1','Oreo'),('2','Hoosier HIll Farm'),('3','Tea's Tea'),('4','Smart Ones'),('5','Green Chile'),('6','Vita'),('7','Cut Russet'),('8','Yoplait'),('9','Sanpellegrino'),('10','Minute Maid'),('11','Pepperidge Farm '),('12','Coca Cola'),('13','Cheezit'),('14','Santa Cruz'),('15','Fatworks'),('16','Suprema'),('17','Urban Grain'),('18','Alo Enrich'); [42000-199]
2019-03-23 21:07:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "VENDOR_ID" not found; SQL statement:
INSERT INTO store (vendor_id, vendor_name, num, street, city, state, zip, account_number) VALUES('1','Walmart','1200','Marketplace Dr','Rochester','NY','14623','3928472819'),('2','Costco','517','E 117th St','New York','NY','10035','2847563627'),('3','Wegmans','3585','California St','San Francisco','CA','94118','2736352637'),('4','Target','250','Fortune Blvd','Milford','MA','1757','6872648571'); [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:708)
	at org.h2.command.Parser.parseColumn(Parser.java:1377)
	at org.h2.command.Parser.parseColumnList(Parser.java:1362)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1731)
	at org.h2.command.Parser.parseInsert(Parser.java:1684)
	at org.h2.command.Parser.parsePrepared(Parser.java:891)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at Database.Populate.VendorTable.populateVendorTable(VendorTable.java:66)
	at Database.RetailMain.main(RetailMain.java:70)
2019-03-23 21:10:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "VENDOR_ID" not found; SQL statement:
INSERT INTO store (vendor_id, vendor_name, num, street, city, state, zip, account_number) VALUES('1','Walmart','1200','Marketplace Dr','Rochester','NY','14623','3928472819'),('2','Costco','517','E 117th St','New York','NY','10035','2847563627'),('3','Wegmans','3585','California St','San Francisco','CA','94118','2736352637'),('4','Target','250','Fortune Blvd','Milford','MA','1757','6872648571'); [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:708)
	at org.h2.command.Parser.parseColumn(Parser.java:1377)
	at org.h2.command.Parser.parseColumnList(Parser.java:1362)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1731)
	at org.h2.command.Parser.parseInsert(Parser.java:1684)
	at org.h2.command.Parser.parsePrepared(Parser.java:891)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at Database.Populate.VendorTable.populateVendorTable(VendorTable.java:66)
	at Database.RetailMain.main(RetailMain.java:70)
2019-03-23 21:12:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "VENDOR_ID" not found; SQL statement:
INSERT INTO store (vendor_id, vendor_name, num, street, city, state, zip, account_number) VALUES('1','Walmart','1200','Marketplace Dr','Rochester','NY','14623','3928472819'),('2','Costco','517','E 117th St','New York','NY','10035','2847563627'),('3','Wegmans','3585','California St','San Francisco','CA','94118','2736352637'),('4','Target','250','Fortune Blvd','Milford','MA','1757','6872648571'); [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:708)
	at org.h2.command.Parser.parseColumn(Parser.java:1377)
	at org.h2.command.Parser.parseColumnList(Parser.java:1362)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1731)
	at org.h2.command.Parser.parseInsert(Parser.java:1684)
	at org.h2.command.Parser.parsePrepared(Parser.java:891)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at Database.Populate.VendorTable.populateVendorTable(VendorTable.java:66)
	at Database.RetailMain.main(RetailMain.java:70)
2019-03-23 21:12:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:617)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1571)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:1237)
	at org.h2.jdbc.JdbcStatement.checkClosed(JdbcStatement.java:1211)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:3245)
	at org.h2.jdbc.JdbcResultSet.next(JdbcResultSet.java:132)
	at Database.Populate.CustomerTable.printCustomerTable(CustomerTable.java:109)
	at Database.RetailMain.main(RetailMain.java:82)
2019-03-23 21:12:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "VENDOR_ID" not found; SQL statement:
INSERT INTO store (vendor_id, vendor_name, num, street, city, state, zip, account_number) VALUES('1','Walmart','1200','Marketplace Dr','Rochester','NY','14623','3928472819'),('2','Costco','517','E 117th St','New York','NY','10035','2847563627'),('3','Wegmans','3585','California St','San Francisco','CA','94118','2736352637'),('4','Target','250','Fortune Blvd','Milford','MA','1757','6872648571'); [42122-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:451)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:708)
	at org.h2.command.Parser.parseColumn(Parser.java:1377)
	at org.h2.command.Parser.parseColumnList(Parser.java:1362)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1731)
	at org.h2.command.Parser.parseInsert(Parser.java:1684)
	at org.h2.command.Parser.parsePrepared(Parser.java:891)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:205)
	at Database.Populate.VendorTable.populateVendorTable(VendorTable.java:66)
	at Database.RetailMain.main(RetailMain.java:70)
2019-03-23 21:14:10 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/abhay/IdeaProjects/Retail_Database/retailDb/retailDb.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:90)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2787)
	at org.h2.mvstore.MVStore.panic(MVStore.java:441)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:404)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:3343)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:95)
	at org.h2.engine.Database.getPageStore(Database.java:2739)
	at org.h2.engine.Database.open(Database.java:769)
	at org.h2.engine.Database.openDatabase(Database.java:319)
	at org.h2.engine.Database.<init>(Database.java:313)
	at org.h2.engine.Engine.openSession(Engine.java:69)
	at org.h2.engine.Engine.openSession(Engine.java:201)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:178)
	at org.h2.engine.Engine.createSession(Engine.java:161)
	at org.h2.engine.Engine.createSession(Engine.java:31)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:336)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:169)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:148)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at Database.RetailMain.createConnection(RetailMain.java:24)
	at Database.RetailMain.main(RetailMain.java:62)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/abhay/IdeaProjects/Retail_Database/retailDb/retailDb.mv.db [1.4.199/7]" [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	... 26 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/abhay/IdeaProjects/Retail_Database/retailDb/retailDb.mv.db [1.4.199/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:883)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:390)
	... 20 more
